include/clangmetatool/meta_tool_factory.h

```diff
@@ -2,14 +2,14 @@
 #define INCLUDED_CLANGMETATOOL_META_TOOL_FACTORY_H
 
 #include <string>
+#include <typeinfo>
```

Comment:
Is this include still needed? should it be in meta_tool.h?

Reply:
It is not longer necessary. Removing it.